lib/entry/LiveEntry.js

```diff
@@ -269,4 +271,57 @@ LiveEntry.prototype.cleanup = function() {
         });
 };
 
+LiveEntry.prototype._registerToPush = function()
+{
+    const pushMgr = new pushManager.PushManager();
+    this.logger.debug(`Registering partner ${this.entryObject.partnerId} for push with id [${this.entryObject.entryId}]`);
+    pushMgr.registerEntry(this.entryObject.entryId, this.entryObject.partnerId, config.get('explicitLivePushNotificationTemplateName'), (...args) => this._pushMessageReceived(...args));
+}
+
+LiveEntry.prototype._pushMessageReceived = function(entry)
+{
+        this.logger.debug(`handling message for entry ${util.inspect(entry)}`);
+        //logger.debug(`@@NA entry [${util.inspect(entry)}]`);
+        //if (entry.viewMode == kalturaTypes.KalturaViewMode.ALLOW_ALL)// @@NA change after client update
+        this._updateEntryObject(entry);
+
+        if (entry.viewMode == 1)
```

Comment:
i think it's better if we call those function only if the state has changed and not redo the action